src/disclosure-policy/verifyAuthorizationAttestation.ts

```diff
@@ -0,0 +1,103 @@
+import { type AgentContext, JwsService, Jwt, SdJwtVcService, type X509Certificate } from '@credo-ts/core'
+import { z } from 'zod'
+import { isRegistrationCertificate } from '../registration-certificate/verifyRegistrationCertificate'
+
+// TODO: support multiple authorization attestation formats.
+//       Currently, only sd-jwt is defined
+const authorizationAttestationHeaderSchema = z.object({
+  alg: z.string(),
+  typ: z.literal('dc+sd-jwt'),
+  x5c: z.array(z.string()),
+})
+
+// TODO:
+//    - Should we support more hashing confirmation claim values?
+const authorizationAttestationPayloadSchema = z.object({
```

Comment:
It would be great if we don't have to redefine the JWT payload every time 🤔 . Fine for now, but might be interesting to build on top of a base defined in e.g. openid4vc libs